MediaBrowser.Api/Images/ImageService.cs

```diff
@@ -723,12 +706,14 @@ private bool SupportsFormat(IEnumerable<string> requestAcceptTypes, string accep
         {
             var mimeType = "image/" + format;
 
-            if (requestAcceptTypes.Contains(mimeType))
+            var requestAcceptTypesList = requestAcceptTypes.ToList();
```

Comment:
Maybe change the function to accept  an `IReadOnlyCollection` instead of copying the `IEnumerable` to a list